exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,15 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(){
+  if(isNegative=-10){
+    return true;
+  }
+    return false;
+}
```

Comment:
Heeey, Ezgi first of all I appreciate your work well done! As we need to review someone code, this time I select your one, and let me add some more points. Here is a task function that return numbers negatives